Bug: Unity Editor Compilation error in EditorExtension.cs

Hi,
I've received this error after copying Ceras to my Unity game project:

Assets/Vendor/CerasSerializer/Ceras.UnityAddon/Editor/EditorExtension.cs(30,23): error CS7036: There is no argument given that corresponds to the required formal parameter 'output' of 'Generator.Generate(string, IEnumerable<Assembly>, StringBuilder)'

I'm assuming it's happening because of this [change](https://github.com/rikimaru0345/Ceras/commit/603a32395c4772aa3ac6e05490b0f77de5aaff0e#diff-fcfbb96cb90db29deb312d613f7c1f773a9df007827198b6dc841346e751bc2aR35).

So I guess `Generator.Generate(assemblies, sb)` needs to be changed to `Generator.Generate(ns, assemblies, sb);`
(and in Ceras.AotGeneratorApp/Program.cs as well).

Can ns be any arbitrary string, or does it need to match my own games namespace?

--- a/src/Ceras.AotGeneratorApp/Program.cs
+++ b/src/Ceras.AotGeneratorApp/Program.cs
@@ -27,7 +27,7 @@ namespace CerasAotFormatterGenerator
 
 			var asms = inputAssemblies.Select(Assembly.LoadFrom);
 			StringBuilder fullCode = new StringBuilder(25 * 1000);
-			Generator.Generate(asms, fullCode);
+			Generator.Generate("AotSerialization", asms, fullCode);
 
 			Console.WriteLine($"Saving...");
 

--- a/src/Ceras.UnityAddon/Editor/EditorExtension.cs
+++ b/src/Ceras.UnityAddon/Editor/EditorExtension.cs
@@ -27,7 +27,7 @@ namespace Ceras.Editor
 
 			// Generate source-code for all formatters
 			var sb = new StringBuilder(25 * 1000);
-			Generator.Generate(assemblies, sb);
+			Generator.Generate("AotSerialization", assemblies, sb);
 			var output = sb.ToString();
 
 			// Check if the output is already exactly the same

